tasks/create-product-sbom/create-product-sbom.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: create-product-sbom
+  labels:
+    app.kubernetes.io/version: "1.0.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Create product-level SBOM from release notes.
+  params:
+    - name: dataJsonPath
+      description: Path to the JSON data.
+      type: string
+  workspaces:
+    - name: data
+      description: Workspace to save the product-level SBOM to.
+  results:
+    - name: productSBOM
+      description: Path to the created product-level SBOM in the workspace.
+  steps:
+    - name: create-sbom
+      # TODO: replace tag with one containing the script
+      image: quay.io/konflux-ci/release-service-utils:35bffc8f17a841a76a247b9ee85ce727f31f7238
+      script: |
+        #!/usr/bin/env sh
+        create_product_sbom --data-path $(params.dataJsonPath) \
```

Comment:
I am a little confused here. The script creates the sbom file in output-path. But it isn't defined what that is.. it just goes to the result. Does this work? It might... I have never tried relying on the results value to be something without explicitly setting it. Just curious

Reply:
I don't see a reason why it wouldn't work. But in general it's not a good idea to store anything bigger and a few lines of text as a result. The limit is 4096 bytes or something like that. I think it would be better to store it in a file in the workspace and pass a relative path to it as a result (like we do for other things).

Reply:
good call, fixed

Reply:
It still needs some tweaks. One, it should use a subdir in the workspace (in the past all the pipeline runs were sharing the same PVC, so they could override each others data; nowadays each PLR has its own PVC, so it's not really an issue anymore, but for now we use the subdir everywhere, so let's stick to it). And second, a relative path in the workspace should be passed instead of an absolute one.

For the subdir, you could pass the subdir as a parameter and use that. But another, perhaps easier, option is to derive it from the data path, e.g.
```sh
SBOM_FILE="product_sbom.json"
OUTPUT_PATH="$(dirname "$(params.dataJsonPath)")/${SBOM_FILE}"

echo -n "$OUTPUT_PATH" > $(results.productSBOMPath.path)
```

BTW, given that the data path is also relative to the workspace, you will also need to adjust the parameter for the `create_product_sbom` call.